src/components/RulesDialog.vue

```diff
@@ -198,7 +198,7 @@
 						Where can I play Cuttle?
 					</h4>
 					<p>
-						You can play Cuttle anywhere you have a deck of cards and a friend! If you want to play online, there is only one place: <a href="www.cuttle.cards">www.cuttle.cards</a>
+						You can play Cuttle anywhere you have a deck of cards and a friend! If you want to play online, there is only one place: <a href="https://www.cuttle.cards">www.cuttle.cards</a>
```

Comment:
While we are at it, would you kindly add the attribute `target=”_blank”` so the link opens in a new window instead of navigating in the current page?